docs/source/api/safe/czml/util.rst

```diff
@@ -0,0 +1,5 @@
+Util module
+===========
+
+.. automodule:: poliastro.czml.util
```

Comment:
> `No module named 'poliastro.czml.util'`

Probably you meant `poliastro.util`?

Reply:
Its a typo, actually I meant poliastro.czml.utils